packages/web5/lib/src/vc/vc_jwt.dart

```diff
@@ -0,0 +1,95 @@
+import 'package:web5/src/jwt.dart';
+import 'package:web5/src/vc/vc.dart';
+
+class DecodedVcJwt {
+  VerifiableCredential vc;
+  DecodedJwt jwt;
+
+  DecodedVcJwt(this.vc, this.jwt);
```

Comment:
do these need to be `required`?

Reply:
i think no, because these are positional parameters not named arguments. If this was `DecodedVcJwt({this.vc, this.jwt});` with `{}` then we would need `required`.